src/pages/HomePage/HomePage.jsx

```diff
@@ -1,45 +1,44 @@
-import { Navbar, Sidebar } from "../../component";
 import "./homePage.css";
-import heroImg4 from "../../Assets/heroImg4.jpg";
-import breakfast from "../../Assets/breakfast.jpeg";
-import lunch from "../../Assets/lunch.jpg";
-import snacks from "../../Assets/snacks.jpg";
-import dinner from "../../Assets/dinner.jpg";
+import { Link } from "react-router-dom";
+
+import heroImg4 from "../../Assets/Images/heroImg4.jpg";
+import breakfast from "../../Assets/Images/breakfast.jpeg";
+import lunch from "../../Assets/Images/lunch.jpg";
+import snacks from "../../Assets/Images/snacks.jpg";
+import dinner from "../../Assets/Images/dinner.jpg";
```

Comment:
you can short these lines... like - export all imgs in one file thn import frm there in single line

`import {img1, img2, img3} from "../../Assets/Images/index.js"`

Reply:
y wanted to do the same but i don't know why it isn't working thats why i came back to the older version of code